src/test/java/com.example.online_shop_api/PasswordServiceTest.java

```diff
@@ -0,0 +1,160 @@
+package com.example.online_shop_api;
+
+import com.example.online_shop_api.Entity.Employee;
+import com.example.online_shop_api.Entity.User;
+import com.example.online_shop_api.Repository.EmployeeRepository;
+import com.example.online_shop_api.Repository.UserRepository;
+import com.example.online_shop_api.Service.PasswordService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class PasswordServiceTest {
+    @InjectMocks
+    private PasswordService passwordService;
+    @Mock
+    private EmployeeRepository employeeRepository;
+    @Mock
+    private UserRepository userRepository;
+    @Mock
+    private BCryptPasswordEncoder bCryptPasswordEncoder;
+    @Mock
+    private MyUserDetails myUserDetails;
+    @Mock
+    private Authentication authentication;
+    private User user;
+    private Employee employee;
+
+    @BeforeEach
```

Comment:
You can remove this setUp() method and use @Mock annotation for the property `private User user; `

Reply:
Done